wp-content/themes/workingnyc/views/employer-programs/archive.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>
+
+    <div class="layout-content">
+      <div class="page-max">
+        <header class="o-header">
+          <div>
+            <nav class="o-header__breadcrumbs" aria-label="Breadcrumb">
+              <a href="/">{{ strings.HOME }}</a>
+
+              <svg aria-hidden="true" class="o-header__breadcrumbs-chevron icon-ui rtl:flip">
+                <use href="#lucide-chevron-right"></use>
+              </svg>
+
+              <b aria-current="page">{{ strings.PAGE_TITLE }}</b>
+            </nav>
+
+            <div class="o-header__title">
+              <h1 id="page-heading" class="o-header__heading">{{ strings.PAGE_TITLE }}</h1>
+            </div>
+
+            <div class="mb-3" v-if="strings.PAGE_CONTENT" v-html="strings.PAGE_CONTENT"></div>
+          </div>
+        </header>
+      </div>
+    </div>
+
+  <div class="grid grid-cols-[1fr_2fr]">
```

Comment:
This is mainly the same as programs/archive.vue, but this div contains both the filters and the program cards, so they are displayed side by side instead of having the filters as a dropdown

<img width="1729" alt="image" src="https://github.com/CityOfNewYork/working-nyc/assets/8957837/97c1147c-9a51-4aa4-8274-01f7145995a9">